server/test/controllers/applicant/UpsellControllerTest.java

```diff
@@ -44,6 +44,7 @@ public void considerRegister_redirectsToUpsellViewForDefaultProgramType() {
     ApplicantModel applicant = createApplicantWithMockedProfile();
     ApplicationModel application =
         resourceCreator.insertActiveApplication(applicant, programDefinition.toProgram());
+    application.setSubmitTimeToNow();
```

Comment:
**question:** Will this cause errors with tests in the future? i.e. should we use a mockclock or something for the time?